tests/local.rs

```diff
@@ -18,3 +18,22 @@ fn test_git_clean_removes_local_branches() {
     assert!(result.stdout().contains("Deleted branch test1"), result.failure_message("command to delete test1"));
     assert!(result.stdout().contains("Deleted branch test2"), result.failure_message("command to delete test2"));
 }
+
+#[test]
+fn test_git_clean_does_not_remove_ignored_local_branches() {
+    let project = project("git-clean_removes_local").build();
+
+    project.setup_command("git branch test1");
+    project.setup_command("git branch test2");
```

Comment:
Can we add a second branch to ignore so we can test the `multi` part of this as well?